config/locales/idv/en.yml

```diff
@@ -185,7 +185,7 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you located near Washington, D.C. <br/>or Baltimore, MD?'
+        are_you_near: 'Are you located near Washington, D.C. <wbr/>or Baltimore, MD?'
```

Comment:
Wondering if it might be worth adding a code comment indicating that there are intentional irregular, non-breaking whitespace in this sentence?